packages/dev/s2-docs/src/CodeBlock.tsx

```diff
@@ -1,5 +1,5 @@
 // @ts-ignore
-import assets from 'url:../pages/**/*.{png,jpg,svg}';
+import assets from 'url:../pages/**/*.{png,jpg,svg}' with {env: 'react-client'};
```

Comment:
Ensuring we only get one copy of the images, not one for the server environment and one for the client environment.